js/data/util/format.js

```diff
@@ -197,10 +201,11 @@ var format = {
    */
   timestamp: function(timestring, offset) {
     var d = new Date(timestring);
+    var f = t('%-I:%M %p');
```

Comment:
Why not having a constant for this one (it is used 2 times in the same file)

Reply:
tests are braking when using the Constant :-(

Reply:
I see. But it may have identify a potential problem.
Maybe theses values are not well translated in blip. Perhaps only if the user change the current language. Since they are only translated the first time the constants.js file is imported.